understanding/22/accessible-authentication-minimum.html

```diff
@@ -72,6 +72,7 @@ <h3>Cognitive Function Tests</h3>
             
             <p>Remembering a site-specific password is a <a>cognitive function test</a>. Such tests are known to be problematic for many people with cognitive disabilities. Whether it is remembering random strings of characters, or a pattern gesture to perform on a touch screen, cognitive function tests will exclude some people. When a cognitive function test is used, at least one other authentication method must be available which is not a cognitive function test.</p>
 
+            <p>Some <abbr title="Completely Automated Public Turing test to tell Computers and Humans Apart">CAPTCHA</abbr> systems have an audio alternative of the visible text. If the user needs to transcribe this audio, it cannot be used to meet the Alternative exception.</p>            
```

Comment:
How is CAPTCHA part of authentication? I seem to remember there was a discussion on that on Github or the mailing list. Since CAPTCHA is not used to identify a particular person but just differentiate human vs. automatic capabilities, this seems misplaced here?

Reply:
it's a cognitive test *as part of* an authentication process (rather than a cognitive test in general). but agree, as mentioned before, this is the the weirdness of this particular SC (that really should have been two separate SCs) special-casing things under the mantle of authentication only

Reply:
@detlev, if you're asking this question, maybe we need to do more than just modify this portion of the document. However, my suggestion would be that spawning a new issue asking that question may be a better approach than adding this contextual information -- unless it is your feeling that this changes makes your underlying question _more_ dominant.

Reply:
I'm easy (adding [rendered view ](https://raw.githack.com/w3c/wcag/06f7b71a542627149dd4d8c0b18765409a6553ed/understanding/22/accessible-authentication-minimum.html)for easier review)